3-reveal/migrations/5-IRS/2-Zambia-2019/deploy/create_structure_geo_hierarchy.psql

```diff
@@ -22,16 +22,16 @@ BEGIN
         jurisdictions.geographic_level AS jurisdiction_depth,
         geo_jurisdictions.id AS geo_jurisdiction_id,
         geo_jurisdictions.geographic_level AS geo_jurisdiction_depth,
-        ST_WITHIN(locations.geometry, geo_jurisdictions.geometry) AS geo_strict_within
-    INTO STRICT geo_record
+        public.ST_WITHIN(locations.geometry, geo_jurisdictions.geometry) AS geo_strict_within
```

Comment:
do we have a geoindices on these columns? https://postgis.net/workshops/postgis-intro/indexing.html

Reply:
it's unclear whether `gist` or `sp-gist` would be best, timing might be the only way to understand

Reply:
Yes these are indexed [here](https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/2-transaction_tables/deploy/jurisdictions.psql#L36) and [here](https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/2-transaction_tables/deploy/locations.psql#L41) respectively.

Reply:
have we checked whether using sp-gist changes the performance characteristics?

Reply:
Will investigate that as part of https://github.com/OpenSRP/opensrp-reveal-datawarehouse/issues/16